app/models/ingredient.rb

```diff
@@ -1,3 +1,3 @@
 class Ingredient < ApplicationRecord
-    belongs_to :dish, optional: true
+    #belongs_to :dish, optional: true
```

Comment:
Delete instead of commenting out of things we don't need at the moment.
If you do need to leave something for people to remember, leave a legit comment instead of commented code.